pkg/controller/migcluster/remote_watch.go

```diff
@@ -91,6 +93,31 @@ func StartRemoteWatch(r *ReconcileMigCluster, config remote.ManagerConfig) error
 	return nil
 }
 
+// IsRemoteWatchConsistent checks whether remote watch in memory is consistent with given config
+func IsRemoteWatchConsistent(nsName types.NamespacedName, newConfig *rest.Config) bool {
+	inMemoryWatch := remote.GetWatchMap().Get(types.NamespacedName{
```

Comment:
Ah, nevermind. I guess we're just accessing the watch map from this helper function instead.